Expecto/Logging.fs

```diff
@@ -1005,7 +1018,7 @@ module internal ANSIOutputWriter =
 
   /// Lifecycle: (new T() -> t.init() -> t._ {0,*} -> (t :> IDisposable).Dispose()) {1,*}
   [<Sealed>]
-  type T(origStdOut: TextWriter, origStdErr: TextWriter, sem: obj) =
+  type T(origStdOut: TextWriter, origStdErr: TextWriter, semD: DVar<obj>, colourLevelD: DVar<ColourLevel>) =
```

Comment:
This refactor could probably be continued at some point; it (this instance/type) is still pretty mutable and not cohesive.